_posts/2019-06-09-eslint-v6.0.0-rc.0-released.md

```diff
@@ -7,15 +7,15 @@ tags:
 ---
 # ESLint v6.0.0-rc.0 released
 
-We just pushed ESLint v6.0.0-rc.0, which is a minor release upgrade of ESLint. This release adds some new features and fixes several bugs found in the previous release.
-
-
-
+We just pushed ESLint v6.0.0-rc.0, which is a new prerelease version for ESLint v6. This release adds some new features and fixes several bugs found in the previous release. This release also has some breaking changes, so please read the following closely.
 
 ## Highlights
 
 This is a summary of the major changes you need to know about for this version of ESLint.
 
+1. Now that BigInt and dynamic import are at stage 4, we have begun to accept PRs to incorporate them into core ESLint rules. The [`valid-typeof`](/docs/rules/valid-typeof) rule now supports BigInt. Please see https://github.com/eslint/eslint/issues/11803 to track our progress in fully supporting these newly standardized language features.
```

Comment:
```suggestion
1. Now BigInt and dynamic import are at stage 4, so we have begun to accept PRs to incorporate them into core ESLint rules. The [`valid-typeof`](/docs/rules/valid-typeof) rule now supports BigInt. Please see https://github.com/eslint/eslint/issues/11803 to track our progress in fully supporting these newly standardized language features.
```

Reply:
Strictly speaking, BigInt is very close to Stage 4; it has gotten consensus on the last TC39 meeting. It will arrive at Stage 4 after the spec editor approved the PR on tc39/ecma262 repo.

Reply:
@g-plane I think both constructions are equally grammatically valid, but feel free to open a PR if you feel strongly.

@mysticatea Thanks, will tweak.

Reply:
@g-plane I figured out an alternative construction that is pretty similar to what you suggested-- hope it's okay.

@mysticatea I've requested your review-- hopefully I've made things accurate without going into too much detail. Thanks!